kayak_core/src/widget.rs

```diff
@@ -120,3 +120,23 @@ where
 }
 
 impl<T> SealedWidget for T where T: Widget {}
+
+impl WidgetProps for () {
```

Comment:
We could also replace this with a concrete `EmptyProps` type if preferred.

Reply:
This is great! 🙂